ngraph/core/src/op/split.cpp

```diff
@@ -222,14 +222,11 @@ namespace
                         const int64_t num_splits,
                         const Node* split_node)
     {
-        int64_t axis;
         switch (axis_tensor->get_element_type())
```

Comment:
According to spec, axis can be any integer number (only i32, i64, u64 was used to reduce binary size) so please replace it with `is_integral_number()` assert.

Reply:
I see that split has evaluate() so binary size concern is valid. Why remove it if it was done on purpose?

Reply:
But it doesn't affect binary size (or I miss sth) so why not ;)
I've tested such case for Split op (Ubuntu, Release, libngraph.so)
1. before using `host_tensor_2_vector`: 7772296
2. Using switch to to accept only i32, i64, u64 as in current version: 7768200
3. My suggestion to remove switch and add assert to accept each integral_number: also 7772296
but correct me if I'm wrong

Reply:
OK, if we don't increase binary size then that's definitely good advise. Let's go with is_integral_number().